lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
Without underscore.js supporting a proper plugin interface, I think it's an anti-pattern to dynamically throw libraries onto eachother's namespaces, because 1) it can be misleading to developers and 2) there can be collisions as noted above.